pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -862,105 +862,117 @@ func TestPriorityQueue_Pop(t *testing.T) {
 }
 
 func TestPriorityQueue_Update(t *testing.T) {
-	objs := []runtime.Object{highPriorityPodInfo.Pod, unschedulablePodInfo.Pod, medPriorityPodInfo.Pod}
 	c := testingclock.NewFakeClock(time.Now())
 	logger, ctx := ktesting.NewTestContext(t)
```

Comment:
Better to move into t.Run?

Reply:
Need to leave them here. As in some cases, `prepareFunc` may use `c` or `logger`.

Reply:
you should change `prepareFunc` to accept a context. Then you can obtain the logger from the context.